armsrc/lfops.c

```diff
@@ -1539,7 +1797,7 @@ void ReadPCF7931() {
         tries++;
         if (BUTTON_PRESS()) return;
     } while (num_blocks != max_blocks);
-end:
+ end:
```

Comment:
Are you sure this label is needed? I think I removed the need for labels in this function a while back, and you don't seem to have added any goto statement?